packages/config-service/src/services/validationService.ts

```diff
@@ -0,0 +1,95 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { GlobalConfig } from './globalConfig';
+
+export class ValidationService {
+  static startUp(envs: NodeJS.Dict<string>): void {
+    // make sure that mandatory fields are passed
+    Object.entries(GlobalConfig.ENTRIES).forEach(([entryName, entryInfo]) => {
+      if (entryInfo.required) {
+        if (!envs[entryName]) {
+          throw new Error(`${entryName} is a mandatory and the relay can not operate without its value.`);
+        }
+
+        if (entryInfo.type === 'number' && isNaN(Number(envs[entryName]))) {
+          throw new Error(`${entryName} must be a valid number.`);
+        }
+      }
+    });
+
+    // make sure that if OPERATOR_KEY_FORMAT is not specified, the provided OPERATOR_KEY_MAIN is in the DER format
```

Comment:
We now run the tests using ECDSA keys.  I don't think DER format should be a requirement.

Reply:
As far as I understand from [this](https://github.com/hashgraph/hedera-json-rpc-relay/issues/3023) ticket, we assume that if `OPERATOR_KEY_FORMAT` is not defined, the passed operator key must be in `DER` format.

There isn't a correlation between ECDSA and DER/HEX in my opinion. An ECDSA key can be encoded in both formats (DER and HEX).

You're right, we're running the tests using the ECDSA key but we've specified [here](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/server/tests/localAcceptance.env#L5) that it's HEX encoded. Another approach could be using the same key but its DER format, assigning it to `OPERATOR_KEY_MAIN` and completely removing `OPERATOR_KEY_FORMAT` from the `localAcceptance.env`.

I may be wrong, what do you think? Or are you proposing to change the default format from DER to HEX? If it is, let's open a new ticket and triage it.